src/App.svelte

```diff
@@ -95,6 +96,8 @@
     <div class="wrapper">
       {#if $activeRouteStore.resource === "home"}
         <Home />
+      {:else if $activeRouteStore.resource === "login"}
```

Comment:
"login" isn't a resource, but an action. What you want here is "session" (and in all other places)